File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/OnSaveBitmap.java
package ja.burhanrashid52.photoeditor;

import android.graphics.Bitmap;

/**
 * @author <a href="https://github.com/burhanrashid52">Burhanuddin Rashid</a>
 * @version 0.1.2
 * @since 5/21/2018
 */
public interface OnSaveBitmap {
    void onBitmapReady(Bitmap saveBitmap);

    void onFailure(Exception e);
}

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/OnSaveBitmap.kt
package ja.mhdxbilal007.photoeditor

import android.graphics.Bitmap

/**
 * @author [Burhanuddin Rashid](https://github.com/mhdxbilal007)
 * @version 0.1.2
 * @since 5/21/2018
 */
interface OnSaveBitmap {
    fun onBitmapReady(saveBitmap: Bitmap)
}